osu.Game/Screens/Ranking/ResultsScreen.cs

```diff
@@ -245,6 +247,12 @@ protected override void Update()
                 addScore(s);
 
             lastFetchCompleted = true;
+
+            if (ScorePanelList.IsEmpty)
```

Comment:
couldn't this just be checking `scores` above?

Reply:
Yes, it can. Well this code is really simple now.

Reply:
Actually no it cannot because the results screen is _paginated_. And scores contains only the _latest fetched page_ of scores. Which leads to this potentially happening:

![osu_2024-01-15_18-39-35](https://github.com/ppy/osu/assets/20418176/8faf16c8-4ba0-41c9-a717-45c921b8c0c3)

I'm reverting the latest commit.

Reply:
Sorry about that. I ran the visual tests and looked at them but missed this...